apps/challenges/models.py

```diff
@@ -102,6 +102,11 @@ def __init__(self, *args, **kwargs):
     is_docker_based = models.BooleanField(
         default=False, verbose_name="Is Docker Based", db_index=True
     )
+    is_static_dataset_docker_based_challenge = models.BooleanField(
+        default=False,
+        verbose_name="Is Static Code Docker Based",
```

Comment:
Change to `Is static dataset docker based`

Reply:
Done.